lib/openHAB/ruleBasedInterpreter.js

```diff
@@ -613,4 +667,9 @@ module.exports = {
     cmd,
     itemLabel,
     itemProperties
+}
+
+if (process.env.NODE_ENV === "test") {
+   module.exports.annotateResult = annotateResult;
+   module.exports.EvaluationResult = EvaluationResult;
```

Comment:
I have to conditionally export some things to test them, I am not very familiar with Node and NPM. If there is a cleaner way to so this I would like to know.

Reply:
I think it is not needed to explicitly check the annotate function. So this can be removed.